src/Request/ParamConverter/UpdateLineItemWebHookParamConverter.php

```diff
@@ -58,7 +58,13 @@ public function apply(Request $request, ParamConverter $configuration): bool
                 (string)$event['eventName'],
                 (string)$event['eventData']['remoteDeliveryId'],
                 (new DateTimeImmutable())->setTimestamp($event['triggeredTimestamp']),
-                $event['eventData']['alias'] ?? null
+                $event['eventData']['alias'],
+                isset($event['eventData']['label']) ?
```

Comment:
why do we need to cast to (string)? if it can be something else you should add validation

$event['eventData']['label'] ?? $event['eventData']['alias']

Reply:
the same about int below